client/apps/game/src/three/scenes/constants.ts

```
@@ -32,137 +90,129 @@ export const castleLevelToRealmCastle: Record<RealmLevels, RealmLevelNames> = {
   [RealmLevels.Empire]: RealmLevelNames.Empire,
 };
 
-export const hyperstructureStageToModel: Record<StructureProgress, string> = {
+export enum StructureProgress {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'StructureProgress'. Consider to delete it or rename it.

'StructureProgress' is defined here:


(lint/suspicious/noRedeclare)